experiments/toptagging/experiment.py

```diff
@@ -271,6 +272,7 @@ def _validate(self, step):
             metrics = self._evaluate_single(
                 self.val_loader, "val", mode="val", step=step
             )
+
```

Comment:
You could remove these extra white lines with 'git checkout -p main file.py', that makes it more readable.